src/index.js

```diff
@@ -16,7 +16,9 @@ function init( {
 			createPopup( popupContainer ),
 		events = customEvents( popup ),
 		showPopup = ( { target } ) => {
-			popup.hide()
+			if ( popup.element.style.visibility === 'visible' ) {
```

Comment:
@hueitan just FYI: I realized tonight that always executing `popup.hide()` without a check was breaking the previews in mobile with the error below (it tries to remove a background screen that's not there), so I'm including this conditional

```
Uncaught TypeError: Node.removeChild: Argument 1 is not an object.
    removeBackgroundScreen webpack://wikipediaPreview/./src/touchPopup.js?:17
    hide webpack://wikipediaPreview/./src/touchPopup.js?:49
    showPopup webpack://wikipediaPreview/./src/index.js?:31
```